sympy/integrals/manualintegrate.py

```diff
@@ -725,10 +725,18 @@ def trig_cotcsc_rule(integral):
                 cotcsc_csceven_condition: cotcsc_csceven
             })((a, b, m, n, integrand, symbol))
 
+def trig_sin2cos_rule(integral):
+    integrand, symbol = integral
+    a = sympy.Wild('a', exclude=[1/(sympy.sin(2*symbol)*sympy.cos(symbol))])
+    match = integrand.match(sympy.sin(2*symbol)*sympy.cos(symbol)*a)
```

Comment:
Yes, I understand why you included `cos(symbol)`. I think that it is better to have a more general case as possible, unless it does not work properly.
Your PR seems good to me.